ArchiSteamFarm/Bot.cs

```diff
@@ -1026,6 +1026,22 @@ internal sealed class Bot : IDisposable {
 								return await ResponseAdvancedRedeem(steamID, args[1], args[2]).ConfigureAwait(false);
 							}
 
+							goto default;
+						case "!TAKE":
```

Comment:
I don't see any point of defining a new command for exactly the same thing. Why not change existing `!transfer <Modes> <Bot>` and `!transfer <Bots> <Modes> <Bot>` into `!transfer <AppID> <ContextID> <Modes> <Bot>` and `!transfer <Bots> <AppID> <ContextID> <Modes> <Bot>`. There is already `All` mode that would fit perfectly in case we don't want any extra logic when not transfering steam items.